packages/node/src/models/app-instance.ts

```diff
@@ -263,7 +264,8 @@ export class AppInstance {
     } catch (e) {
       // TODO: Catch ethers.errors.INVALID_ARGUMENT specifically in catch {}
       console.error(
-        "Attempted to setState on an app with an invalid state object"
+        "Attempted to setState on an app with an invalid state object",
```

Comment:
You can easily format this string to include the app ID, the invalid object and the state encoding that it should adhere to.